plugins/snowflake/dbt/include/snowflake/macros/adapters.sql

```diff
@@ -1,14 +1,33 @@
 {% macro snowflake__create_table_as(temporary, relation, sql) -%}
   {%- set transient = config.get('transient', default=true) -%}
+  {# TODO: Add support for more than one keys #}
+  {%- set cluster_by_keys = config.get('cluster_by') -%}
```

Comment:
I like using a pattern like [this](https://github.com/fishtown-analytics/dbt/blob/e46800f5b4be07b23989594f9a110b75e89b7867/plugins/redshift/dbt/include/redshift/macros/adapters.sql#L16-L28) which allows users to either specify:
1. a string
2. a list of strings

It's a tiny bit of extra work to marshal a string into a list that contains one string, but it's worth the UX benefit IMO!

Reply:
It's actually way more elegant that what I had done so far (🤦‍♂ ) in terms of logic which was:
```	  {%- if cluster_by_keys is not string and cluster_by_keys is not none -%}
    {%-  set cluster_by_keys = cluster_by_keys|join(", ") -%}
  {%- endif -%}
```